app/Nova/User.php

```diff
@@ -148,6 +148,19 @@ public function fields(NovaRequest $request): array
 
             HasMany::make('Signatures'),
 
+            new Panel(
```

Comment:
Is a panel with no visible fields hidden?

Reply:
Yes